compiler-test/build.gradle

```diff
@@ -0,0 +1,49 @@
+import org.gradle.internal.jvm.Jvm
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
+
+    defaultConfig {
+        minSdkVersion 15
+        targetSdkVersion 25
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    testCompile project(":compiler")
+    testCompile project(":library")
+
+    testCompile 'junit:junit:4.12'
+    testCompile 'commons-io:commons-io:2.4'
+    testCompile 'com.google.testing.compile:compile-testing:0.6'
+    testCompile files(getRuntimeJar())
+}
+
+
+def getRuntimeJar() {
```

Comment:
I found that we can't use `javax.tools.JavaFileObject` without `rt.jar` in android library project.
Though I didn't take a deep look at this, maybe this class is not included in Android's java runtime?
So I added for testing purpose.